pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "urdfenvs"
-version = "0.1.8"
```

Comment:
Nice, I see automatic version control works!

Reply:
I did it manually :(